docs/book/programs/xdp.md

```diff
@@ -1,3 +1,482 @@
 # XDP
 
-This page is a work in progress, please feel free to open a Pull Request!
+!!! example "Source Code"
+
+    Full code for the example in this chapter is available [here](https://github.com/aya-rs/book/tree/main/examples/xdp-drop)
+
+
+## What is XBP in eBPF?
+
+XBP (eXpress Data Path) is a type of eBPF program that attaches to the network interface. It allows for the processing of network packets as soon as they are received from the network driver, even before they enter the networking stack. XDP enables efficient packet filtering, manipulation and redirection at the earliest possible stage, resulting in low-latency and high-throughput.
```

Comment:
* Typo (s/XBP/XDP/)
* The last sentence seems a bit redundant. I think it would be better to merge it with the previous one.

My suggestion:

```suggestion
XDP (eXpress Data Path) is a type of eBPF program that attaches to the network interface. It enables filtering, manipulation and refirection of network packets as soon as they are received from the network driver, even before they enter the Linux kernel networking stack, resulting in low latency and high throughput.
```